src/TimeModeling/judiWeights.jl

```diff
@@ -114,7 +116,7 @@ end
 function +(a::judiWeights{avDT},b::Number) where avDT
     c = deepcopy(a)
     for j=1:a.nsrc
-        c.weights[j] = c.weights[j] .+ b
+        c.weights[j] = convert(Array{avDT}, c.weights[j] .+ b)
```

Comment:
conversion are bad and expensive, why is this needed now, if any is needed it should be on `b` only, this makes an extra copy for the conversion.

Reply:
Previously, Float64 number * Float32 judiWeights could generate a Float64 judiWeights, which we want to avoid. And I agree with you, I will only do the conversion for b then!